packages/framework/src/Component/Cron/CronModuleFacade.php

```diff
@@ -141,7 +141,7 @@ public function markCronAsFailed(CronModuleConfig $cronModuleConfig): void
         $cronModule = $this->cronModuleRepository->getCronModuleByServiceId($cronModuleConfig->getServiceId());
         $cronModule->setStatusFailed();
 
-        $this->em->flush($cronModule);
+        $this->em->flush();
```

Comment:
method `markCronAsFailed()` is called on any throwable and this flush() without parameter can save some invalid state of entities. Is it safe and acceptable?

Reply:
true indeed, rewritten in 63d6f7432